doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -149,6 +149,16 @@ nrf9160 Samples
       * An option ``--interval`` (in seconds) to neighbor cell measurements in continuous mode  (``link ncellmeas --continuous``).
         When using this option, a new measurement is started in each interval.
 
+Zigbee samples
+--------------
+
+* Updated documentation with newly introduced factory reset functionality in:
```

Comment:
```suggestion
* Updated documentation with newly introduced factory reset functionality in the following:
```

I believe it needs to be like this, that we cannot just say `in: ` and then list. But perhaps I am wrong. In that case, disregard my comment. @greg-fer can confirm?